Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.10-alpine3.16
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+# RUN apt-get update -y && apt-get upgrade -y
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN pip3 install --upgrade pip && pip3 install -r requirements.txt
+COPY . .
+EXPOSE 5000
+CMD ["python3", "app.py"]
```

Comment:
consider using [gunicorn](https://github.com/benoitc/gunicorn) for production deployment

Reply:
Yeah, for the purpose of simplicity and making HW done as quickly as possible, there were no gunicorn, uwsgi, nginx, etc implemented. But in real world those technologies are must have.